.gitignore

```diff
@@ -2,3 +2,10 @@
 default.profraw
 _site
 .jekyll-metadata
+
+# npm
+node_modules/
+*/package-lock.json
+
+#sass
+_sass/glue.scss
```

Comment:
I didn't add this file, so I am assuming it was auto generated and needs to be added to the ignore. Please confirm.

Reply:
Nack. I removed this file in #483. You may need to merge master in to your PR, perhaps?

Reply:
Weird. Admittedly, I'm merging this across multiple repos and configurations and merging is quite a pain. Something I suspect will manifest more prominently once we separate the AIP content from the AIP infra.